src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,43 @@
+import { Link, useParams } from "react-router-dom";
+
+import type { EntityDetailsRoute } from "components/Routes/Routes";
+import React from "react";
+
+export default function Breadcrumb(): JSX.Element {
+  const { userName, modelName, appName } = useParams<EntityDetailsRoute>();
+
+  const generateModelURL = function (): string {
+    if (userName) {
+      return `/models/${userName}/${modelName}`;
+    } else {
+      return `/models/${modelName}`;
+    }
+  };
+
+  return (
+    <nav className="p-breadcrumbs" aria-label="Breadcrumb navigation">
+      <ol className="p-breadcrumbs__items" data-test="breadcrumb-items">
+        {appName ? (
+          <>
+            <li className="p-breadcrumbs__item" data-test="breadcrumb-model">
+              <Link to={generateModelURL()}>{modelName}</Link>
+            </li>
+            <li className="p-breadcrumbs__item" data-test="breadcrumb-section">
+              <Link to={generateModelURL()}>Applications</Link>
```

Comment:
I find it strange that the only two clickable breadcrumbs both link to the same place?

Reply:
This is per design but it'll change as more pages are implemented into the breadcrumb.

Reply:
I agree, and design may change there mind now its in practise.